### PR TITLE
Upgrading jackson from 1.6.2 to 1.9.13

### PR DESCRIPTION
**Pull request by i004ncc**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

File: dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java [hunk 1]
```diff
@@ -58,7 +60,9 @@ public JSONJacksonImpl(JSONSettings settings) {
     @Override
     public String stringify(Object o) throws JSONMarshallException {
         try {
-            return mapper.writeValueAsString(o);
+            String s = mapper.writeValueAsString(o);
+            System.out.println("The value from stringify is" + s);
```
Comment (WhiteCat22) [inline dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java]: You'll want to remove this or make it a debug statement :)

File: dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java [hunk 2]
```diff
@@ -48,8 +48,10 @@ public JSONJacksonImpl(JSONSettings settings) {
         //set inclusion
         Include inclusion = settings.getInclusion();
         if (inclusion == Include.ALWAYS) {
+            System.out.println("=====    Inclusion is Always    =====");
```
Comment (WhiteCat22) [inline dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java]: You'll want to remove this or make it a debug statement :)

File: dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java [hunk 3]
```diff
@@ -48,8 +48,10 @@ public JSONJacksonImpl(JSONSettings settings) {
...
             mapper.getSerializationConfig().setSerializationInclusion(Inclusion.ALWAYS);
         } else if (inclusion == Include.NON_NULL) {
+            System.out.println("=====    Inclusion is Non Null    =====");
```
Comment (WhiteCat22) [inline dev/io.openliberty.jsonsupport.internal/src/com/ibm/ws/jsonsupport/internal/JSONJacksonImpl.java]: You'll want to remove this or make it a debug statement :)

**Review by WhiteCat22 — CHANGES_REQUESTED** [review]

**Review by WhiteCat22 — DISMISSED** [review]

File: dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository.xml [hunk 4]
```diff
@@ -233,11 +233,11 @@
             <version-range>[1.0.0,1.1.0)</version-range>
         </constituent>
         <constituent>
-            <symbolic-name>com.ibm.websphere.jsonsupport</symbolic-name>
+            <symbolic-name>io.openliberty.jsonsupport.internal</symbolic-name>
             <start-level>12</start-level>
             <activation-type>SEQUENTIAL</activation-type>
             <type>type=osgi.bundle</type>
-            <version-range>[1.0.0,1.1.0)</version-range>
+            <version-range>[2.0.0,3.0.0)</version-range>
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository.xml]:
```suggestion
            <version-range>[1.0.0,1.1.0)</version-range>
```
The bundle version is still 1.0

File: dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository.xml [hunk 5]
```diff
@@ -50985,11 +50985,11 @@
...
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository.xml]:
```suggestion
            <version-range>[1.0.0,1.1.0)</version-range>
```
The bundle version is still 1.0

File: dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository_WL.xml [hunk 6]
```diff
@@ -233,11 +233,11 @@
             <version-range>[1.0.0,1.1.0)</version-range>
         </constituent>
         <constituent>
-            <symbolic-name>com.ibm.websphere.jsonsupport</symbolic-name>
+            <symbolic-name>io.openliberty.jsonsupport.internal</symbolic-name>
             <start-level>12</start-level>
             <activation-type>SEQUENTIAL</activation-type>
             <type>type=osgi.bundle</type>
-            <version-range>[1.0.0,1.1.0)</version-range>
+            <version-range>[2.0.0,3.0.0)</version-range>
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository_WL.xml]:
```suggestion
            <version-range>[1.0.0,1.1.0)</version-range>
```
The bundle version is still 1.0

File: dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository_WL.xml [hunk 7]
```diff
@@ -87267,11 +87267,11 @@
...
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.feature.resolver_fat/publish/verify/repository_WL.xml]:
```suggestion
            <version-range>[1.0.0,1.1.0)</version-range>
```
The bundle version is still 1.0

File: dev/com.ibm.ws.ui/main.bnd [hunk 8]
```diff
@@ -28,7 +28,7 @@ Private-Package: \
 
 # Need a special import to properly construct import for commons.io
 Import-Package: \
-  com.ibm.websphere.jsonsupport.*;version='[1.0,2.0)', \
+  io.openliberty.jsonsupport.internal.*;version='[2.0,3.0)', \
```
Comment (jhanders34) [inline dev/com.ibm.ws.ui/main.bnd]:
```suggestion
  com.ibm.websphere.jsonsupport.*;version='[2.0,3.0)', \
```
The exported package name did not change.

**Review by jhanders34 — CHANGES_REQUESTED** [review]

**Comment by i004ncc** [comment]
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=329ee55a-7b18-4f9c-9631-d838ca396607) - :x: completed with errors/failures.) ([view Open Liberty Personal Build RTC](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=6b50991b-7bd4-416f-a208-5fbb82874195) - :x: completed with errors/failures.)
git.clone.checkout.branch=upgrade_jackson_1x_WS_CD_part
git.clone.repository.sshUrl=git@github.ibm.com:NANDAKRISHNANPN/WS-CD-Open.git
git.clone.repository.username=NANDAKRISHNANPN
fat.buckets.to.run=all

_Note: Target locations of links might be accessible only to IBM employees._

_Note: Target locations of links might be accessible only to IBM employees._

**Review by jhanders34 — APPROVED** [review]

**Comment by i004ncc** [comment]
!build
 ([view Open Liberty Personal Build RTC](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=f108ffb2-017f-4320-8bbd-46cca256c10e) - :x: completed with errors/failures.) ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=92453e34-a2ee-41b0-aa25-ec6e208e3101) - :x: completed with errors/failures.)
git.clone.checkout.branch=upgrade_jackson_1x_WS_CD_part
git.clone.repository.sshUrl=git@github.ibm.com:NANDAKRISHNANPN/WS-CD-Open.git
git.clone.repository.username=NANDAKRISHNANPN
fat.buckets.to.run=all

_Note: Target locations of links might be accessible only to IBM employees._

_Note: Target locations of links might be accessible only to IBM employees._

**Pull request merged**